app/assets/stylesheets/govuk_publishing_components/components/print/_accordion.scss

```diff
@@ -1,28 +1 @@
-// This is in place until a proper fix comes through from GOV.UK Frontend.
-//
-// TODO: Replace with the print styles that will come from GOV.UK Frontend.
-
-@import "govuk_publishing_components/component_support";
```

Comment:
Just checking - is it okay that this `import` isn't being copied over as well?

Reply:
Thanks James, it was initially copied over as well, but after further investigation it is not required.

Further details are included on this comment: https://github.com/alphagov/govuk_publishing_components/pull/3114#discussion_r1040929056

Reply:
Cool, thanks Martin! 👍